From fa84d24dbc4249717bd63fcada536d8216107b9c Mon Sep 17 00:00:00 2001
From: John Greene <jogreene@redhat.com>
Date: Mon, 8 Sep 2014 15:08:03 -0400
Subject: [ethernet] ixgbevf: Remove unused get_supported_physical_layer pointer

Message-id: <1410188883-6595-49-git-send-email-jogreene@redhat.com>
Patchwork-id: 92360
O-Subject: [RHEL7.1 PATCH 48/48] ixgbevf: Remove unused get_supported_physical_layer pointer
Bugzilla: 1091124
RH-Acked-by: Neil Horman <nhorman@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>
RH-Acked-by: John Linville <linville@redhat.com>

BZ: 1091124
Brew: https://brewweb.devel.redhat.com/taskinfo?taskID=7919168
Tested: In beaker. See BZ for details

Signed-off-by: Mark Rustad <mark.d.rustad@intel.com>
Tested-by: Phil Schmitt <phillip.j.schmitt@intel.com>
Signed-off-by: Jeff Kirsher <jeffrey.t.kirsher@intel.com>
(cherry picked from commit 9f1fb8acd30c9ace0145e66942481bdb90beca15)
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/intel/ixgbevf/vf.h b/drivers/net/ethernet/intel/ixgbevf/vf.h
index 3061d18..aa8cc8d 100644
--- a/drivers/net/ethernet/intel/ixgbevf/vf.h
+++ b/drivers/net/ethernet/intel/ixgbevf/vf.h
@@ -49,7 +49,6 @@ struct ixgbe_mac_operations {
  s32 (*start_hw)(struct ixgbe_hw *);
  s32 (*clear_hw_cntrs)(struct ixgbe_hw *);
  enum ixgbe_media_type (*get_media_type)(struct ixgbe_hw *);
- u32 (*get_supported_physical_layer)(struct ixgbe_hw *);
  s32 (*get_mac_addr)(struct ixgbe_hw *, u8 *);
  s32 (*stop_adapter)(struct ixgbe_hw *);
  s32 (*get_bus_info)(struct ixgbe_hw *);
-- 
1.7.1